lib/eclipse/Deck/DeckKeyword.cpp

```diff
@@ -17,9 +17,9 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "DeckKeyword.hpp"
```

Comment:
Easy enough to split out when things are ready, and easier for me to work with one PR. Making it public has mostly to do with travis.